niviz/interfaces/views.py

```diff
@@ -48,3 +50,75 @@ def _post_run_hook(self, runtime):
 
     def _run_interface(self, runtime):
         return runtime
+
+
+class _ISegInputSpecRPT(nrc._SVGReportCapableInputSpec,
+                        BaseInterfaceInputSpec):
+    '''
+    Input specification for ISegRPT, implements:
+
+    anat_file: Input anatomical image
+    seg_files: Input segmentation image(s) - can be a list or a single file
+    mask_file: Input ROI mask
+
+    Bases _SVGReportCapableInputSpec which implements:
+
+    out_report: Filename trait
+    compress_report: ["auto", true, false]
+
+    '''
+    anat_file = File(exists=True,
+                     usedefault=False,
+                     resolve=True,
+                     desc='Anatomical image of SVG',
+                     mandatory=True)
+
+    seg_files = InputMultiPath(
+                    File(exists=True,
+                         usedefault=False,
+                         resolve=True),
+                    desc='Segmentation image of SVG',
+                    mandatory=True)
+
+    mask_file = File(exists=True,
+                     udedefault=False,
+                     resolve=True,
+                     desc='ROI Mask for mosaic',
+                     mandatory=True)
```

Comment:
Just took a look at `plot_segs()` and mask_file is optional. I think what happens is that when setting an input spec as `mandatory=False`, nipype expects you to deal with it. I'm sure there's a more conventional way using `traits` but i don't know it off the top of my head

`masked` can be an optional `traits.Bool` flag!

```suggestion
    mask_file = File(exists=True,
                     resolve=True,
                     desc='ROI Mask for mosaic')
                     
    masked = traits.Bool(False,
                     usedefault=True,
                     desc='Flag to indicate whether'
                     ' image is already masked')
```